docs/asyncapi.yaml

```diff
@@ -0,0 +1,126 @@
+asyncapi: 3.0.0
+info:
+  title: API contract book for Single Front Door user communications
+  version: 1.0.0
+  description: Publish communications to users via the Single Front Door
+
+channels:
+  fcp-fd-comms:
+    address: fcp-fd-comms-request
+    description: |
+      Accepts requests for communication notifications via Azure service bus.
+    messages:
+      notifyMessage:
+        $ref: '#/components/messages/notifyMessage'
+
+operations:
+  SourceSystem/fcp-fd-comms.subscribe:
+    action: receive
+    channel:
+      $ref: '#/channels/fcp-fd-comms'
+    
+
+components:
+  messages:
+    notifyMessage:
+      title: gov.notify communications message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/govNotifyMessage'
+
+  schemas:
+    govNotifyMessage:
```

Comment:
I think we should drop govNotify from the name.  I'd like us to avoid tight coupling the contract to a tech choice.  It may not always be Notify that handles these messages

Reply:
I thought we would have a schema per each destination aka if we going to be using a different provider it's safe to assume that they will have there own unique message structure, thus we would need a new schema, however now I am thinking that we can just normalise existing message in future if we will have a different provider. Renamed to userNotificationMessage